test/primer/hyperloglog_test.cpp

```diff
@@ -84,39 +83,40 @@ TEST(HyperLogLogTest, DISABLED_BasicTest2) {
     obj.AddElem(-7);
     obj.AddElem(-8);
     obj.AddElem(-9);
+    obj.AddElem(-27);
     if (i == 0) {
       obj.ComputeCardinality();
       ans = obj.GetCardinality();
-      ASSERT_EQ(ans, 15);
+      ASSERT_EQ(ans, 10);
     }
   }
   obj.ComputeCardinality();
   ans = obj.GetCardinality();
-  ASSERT_EQ(ans, 15);
+  ASSERT_EQ(ans, 10);
 }
 
 TEST(HyperLogLogTest, DISABLED_EdgeTest1) {
-  auto obj1 = HyperLogLog<int64_t>(-2);
+  auto obj1 = HyperLogLog<int64_t>(static_cast<int16_t>(-2));
   obj1.ComputeCardinality();
   ASSERT_EQ(obj1.GetCardinality(), 0);
 }
 
 TEST(HyperLogLogTest, DISABLED_Edgetest2) {
```

Comment:
This needs to change to `EdgeTest2` with a capital, I changed it in the private version